.circleci/config.yml

```diff
@@ -86,8 +86,8 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - spree-bundle-v3-{{ checksum ".ruby-version" }}-{{ .Branch }}
-            - spree-bundle-v3-{{ checksum ".ruby-version" }}
+            - spree-bundle-v4-{{ checksum ".ruby-version" }}-{{ .Branch }}
```

Comment:
Shouldn't we add `spree-bundle-v4-{{ checksum ".ruby-version" }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}` hey here?